tools/cockpit.spec

```diff
@@ -160,7 +160,13 @@ make -j4 %{?extra_flags} all
 
 %check
 exec 2>&1
-make -j4 check
+# HACK: Fedora koji builders are very slow, unreliable, and inaccessible for debugging; https://github.com/cockpit-project/cockpit/issues/13909
+%if 0%{?fedora} >= 0
+%ifarch s390x
+%define testsuite_fail || true
+%endif
+%endif
+make -j4 check %{?testsuite_fail}
```

Comment:
nice, so `testsuite_fail` is defined for fedora on s390x as `|| true` and otherwise it is undefined. And doing `%{?testsuite_fail}` for undefined `testsuite_fail` will just produce empty string? (Because there is this `0%{?fedora} ` which says that if it is not defined, use `0`, thus wonder if "nothing" is default)

Reply:
Yes, "nothing" is the default, and thus for numeric comparisons you need to prepend an extra 0 to make it syntactically valid. But "empty" is just fine in this case.